### ISSUE TITLE
flux:wizard.link (type page) not working after upgrade to 7.6.5

### ISSUE THREAD

**Issue by cybergwei** [FluidTYPO3/flux#1127]
after upgrade to 7.6.5 flux:wizard.link in combination with flux.field a page link isn't possible. it says its a required field. page type file is working. 

after switching back to to 7.6.4 page link is working again.

**Comment by dachande** [comment]
This bug is also already being discussed in the forge at https://forge.typo3.org/issues/75744
It's still unclear if this is a core bug introduced with TYPO3 7.6.5 or if this bug is flux related, but chances are, that this bug is somehow flux related as the link field used in FAL elements available in core content elements (like the image content element) is not affected.

**Comment by henrysingleton** [comment]
After doing some testing between 7.6.5 and 7.5.4 I have narrowed this down to some changes made in sysext/backend/Resources/Public/JavaScript/FormEngineValidation.js, possibly this commit: https://github.com/TYPO3-CMS/backend/commit/0bdb1e4321af343ecf855a644ffbf29d31fdb2da

Can be reproduced on 7.6.5 by adding any text field to a page configuration, and attempting to enter only a numerical value. The validation will immediately run and invalidate the field, preventing submission of the form. This can also be reproduced on 7.6.4 by replacing FormEngineValidation.js with the one from 7.6.5. 

It seem like the validation JS is acting upon the "range" validation configuration, even though the upper and lower limits are NULL. Removing the "range" config from the fields data attribute makes it behave normally. 

Unfortunately I don't have time to investigate further or provide a fix. For now I have downgraded back to 7.6.4

Note that this affects ANY input field in the backend that contains only a numerical value. This includes any linking field that contains only a page ID, but I also had it occur on a regular input field that just had a year value in it.
